### PR TITLE
Auto-renewal Toggle: Add a "I'll keep it" button to the disabling auto-renewal dialog.

### PR DESCRIPTION
**Pull request by southp**
#### Changes proposed in this Pull Request

This PR adds a "I'll keep it" button to the disabling auto-renewal dialog, which closes the dialog without changing the auto-renewal status. Props to @niranjan-uma-shankar .
image

#### Testing instructions

1. Go to the purchase management page, http://calypso.localhost:3000/me/purchases/,  and open a subscription with auto-renewal on.
1. Turn off the auto-renewal through the toggle. You should see the dialog with a "I'll keep it button".
1. Clicking on the button should close the dialog without changing the auto-renewal status.

### CODE REVIEW

File: client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx [hunk 1]
```diff
@@ -171,6 +171,7 @@ class AutoRenewDisablingDialog extends Component {
 				<Button onClick={ this.onClickGeneralConfirm } primary>
 					{ translate( 'Confirm cancellation' ) }
 				</Button>
+				<Button onClick={ onClose }>{ translate( "I'll keep it" ) }</Button>
```
Comment (a8ci18n) [inline client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx]: Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'I'll Keep It' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/cancel-button-on-toggling-off-dialog)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by southp** [comment]
@michelleweber One thing I need the editor team opinion: Should all the words in a button text starting with capital letters?

I'm asking because I notice that there are many places are doing this like this survey dialog:
image
while in the screenshot above you can see that I've only put capital letters at the beginning.

What is the correct convention here? I'm happy to update my wording for consistency.

**Comment by kristastevens** [comment]
Hi @southp, thanks for the ping! 

> One thing I need the editor team opinion: Should all the words in a button text starting with capital letters?

Button conventions should be like this example: "Button text" where the first word is capitalized, and any remaining words are lowercased. Reference:  [the Dot Com Brand Guide](https://dotcombrand.wordpress.com/style/):

> For headings, buttons, and text in general, we use sentence case.

**Comment by southp** [comment]
Ah, great point, @niranjan-uma-shankar . I've updated it by e3447b9:
image

File: client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx [hunk 2]
```diff
@@ -168,9 +168,12 @@ class AutoRenewDisablingDialog extends Component {
 			>
 				<h2 className="auto-renew-disabling-dialog__header">{ translate( 'Before you go…' ) }</h2>
 				<p>{ description }</p>
-				<Button onClick={ this.onClickGeneralConfirm } primary>
+				<Button onClick={ this.onClickGeneralConfirm }>
 					{ translate( 'Confirm cancellation' ) }
 				</Button>
+				<Button onClick={ onClose } primary>
+					{ translate( "I'll keep it" ) }
```
Comment (a8ci18n) [inline client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx]: Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'I'll Keep It' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/cancel-button-on-toggling-off-dialog)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Pull request merged**